### PR TITLE
Fix Clerk Signin

### PR DESCRIPTION
**Pull request by JJwilkin**

### CODE REVIEW

File: middleware.ts [hunk 1]
```diff
@@ -3,11 +3,11 @@ import { NextRequest, NextResponse } from 'next/server'
...
-  if (!userId) {
-    const signInUrl = new URL('/sign-in', req.url)
-    signInUrl.searchParams.set('redirect_url', req.url)
-    return NextResponse.redirect(signInUrl)
-  }
+  // if (!userId) {
+  //   const signInUrl = new URL('/sign-in', req.url)
+  //   signInUrl.searchParams.set('redirect_url', req.url)
+  //   return NextResponse.redirect(signInUrl)
+  // }
```
Comment (hnagri52) [inline middleware.ts]: this can be left uncommented

File: middleware.ts [hunk 2]
```diff
@@ -2,19 +2,19 @@ import { getAuth, withClerkMiddleware } from '@clerk/nextjs/server'
...
-    const signInUrl = new URL('/sign-in', req.url)
-    signInUrl.searchParams.set('redirect_url', req.url)
-    return NextResponse.redirect(signInUrl)
-  }
+  // const { userId } = getAuth(req)
+  // if (!userId) {
+  //   const signInUrl = new URL('/sign-in', req.url)
+  //   signInUrl.searchParams.set('redirect_url', req.url)
+  //   return NextResponse.redirect(signInUrl)
+  // }
```
Comment (pharmaboxfydp) [inline middleware.ts:10]: I don't think this should be commented still

Comment (hnagri52) [inline middleware.ts]: don't think this should be commented still

**Review by hnagri52 — COMMENTED** [review]
i think you need to fix merge conflicts as well before merging

File: components/UserSignIn.tsx [hunk 3]
```diff
@@ -36,26 +38,18 @@ export default function UserSignIn() {
...
     </Box>
   )
 }
+const SignedOutCards = () => {
+  const { openSignIn, openSignUp } = useClerk()
+  return (
+    <>
+      <a onClick={() => openSignIn()} className={styles.card}>
+        <h2>Sign In</h2>
+        {/* <p>Sign In</p> */}
```
Comment (hnagri52) [inline components/UserSignIn.tsx]: can delete line 51

File: pages/_app.tsx [hunk 4]
```diff
@@ -14,20 +15,32 @@ NProgress.configure({
   showSpinner: false
 })
 
-Router.events.on('routeChangeStart', () => NProgress.start())
-Router.events.on('routeChangeComplete', () => NProgress.done())
-Router.events.on('routeChangeError', () => NProgress.done())
+// Router.events.on('routeChangeStart', () => NProgress.start())
+// Router.events.on('routeChangeComplete', () => NProgress.done())
+// Router.events.on('routeChangeError', () => NProgress.done())
```
Comment (hnagri52) [inline pages/_app.tsx]: can remove these commented lines

**Review by hnagri52 — APPROVED** [review]

**Pull request merged**